x/auth/client/cli/tx_sign.go

```diff
@@ -150,7 +150,7 @@ func setOutputFile(cmd *cobra.Command) (func(), error) {
 }
 
 // GetSignCommand returns the transaction sign command.
-func GetSignCommand(codec *codec.Codec) *cobra.Command {
+func GetSignCommand(clientCtx client.Context) *cobra.Command {
```

Comment:
ditto test

Reply:
This has CLI tests, does it also need a unit test? It didn't before, we're just migrating it here.